deps/db/src/logseq/db/frontend/property.cljs

```diff
@@ -38,6 +38,90 @@
                                    :cardinality :many
                                    :public? true
                                    :classes #{:logseq.class/Root}}}
+   :kv/value             {:title "KV value"
+                          :attribute :kv/value
+                          :schema {:type :any
+                                   :public? false
+                                   :hide? true}}
+   :block/type           {:title "Node type"
+                          :attribute :block/type
+                          :schema {:type :string
+                                   :public? false
+                                   :hide? true}}
+   :block/schema         {:title "Node schema"
+                          :attribute :block/schema
+                          :schema {:type :map
+                                   :public? false
+                                   :hide? true}}
+   :block/parent         {:title "Node parent"
```

Comment:
This seems like an implementation detail that will confuse users e.g. how is it and why is it different than the parent property?